<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator System</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="elevatorGrp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>471</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator System</string>
    </property>
    <widget class="QPushButton" name="helpBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="floorNumberTxtBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="elevatorNumberTxtBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="doorIntBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Interrupt Door</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fireBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire detected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="overLoadBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OverLoad Detected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerOutBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Out detected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReqElevBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Request Elevator</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
